test/client/getOrderbook.ts

```diff
@@ -72,142 +55,136 @@ function normalRippledResponse(
   throw new Error('unexpected end')
 }
 
-// function xrpRippledResponse(
-//   request: BookOffersRequest,
-// ): Record<string, unknown> {
-//   if (request.taker_pays.issuer === 'rp8rJYTpodf8qbSCHVTNacf8nSW8mRakFw') {
-//     return rippled.book_offers.xrp_usd
-//   }
-//   if (request.taker_gets.issuer === 'rp8rJYTpodf8qbSCHVTNacf8nSW8mRakFw') {
-//     return rippled.book_offers.usd_xrp
-//   }
-//   throw new Error('unexpected end')
-// }
+function xrpRippledResponse(
+  request: BookOffersRequest,
+): Record<string, unknown> {
+  if (request.taker_pays.issuer === 'rp8rJYTpodf8qbSCHVTNacf8nSW8mRakFw') {
+    return rippled.book_offers.xrp_usd
+  }
+  if (request.taker_gets.issuer === 'rp8rJYTpodf8qbSCHVTNacf8nSW8mRakFw') {
+    return rippled.book_offers.usd_xrp
+  }
+  throw new Error('unexpected end')
+}
 
 describe('client.getOrderbook', function () {
   beforeEach(setupClient)
   afterEach(teardownClient)
 
-  addressTests.forEach(function (testcase) {
-    describe(testcase.type, function () {
-      it('normal', async function () {
-        this.mockRippled.addResponse('book_offers', normalRippledResponse)
-        const response = await this.client.getOrderbook(
-          requests.getOrderbook.normal.taker_pays,
-          requests.getOrderbook.normal.taker_gets,
-          { limit: 1 },
-        )
-        assert.deepEqual(response, responses.getOrderbook.normal)
-      })
+  it('normal', async function () {
+    this.mockRippled.addResponse('book_offers', normalRippledResponse)
+    const response = await this.client.getOrderbook(
+      requests.getOrderbook.normal.takerPays,
+      requests.getOrderbook.normal.takerGets,
+      {
+        limit: 1,
+      },
+    )
+    const expectedResponse = {
+      buy: responses.getOrderbook.normal.buy.slice(0, 1),
+      sell: responses.getOrderbook.normal.sell.slice(0, 1),
```

Comment:
Flagging this as potentially sneaky spot where it would be good to use the limit defined above instead of new constants.